### DISCUSSION TITLE
troca automatica

### DISCUSSION THREAD

**Discussion by tadkn** [rocksdanister/lively#1372] [General]
bom dia eu resolvi agora pouco instalar o app no meu pc pois nao é a primeira vez que vejo pessoas por exemplo no tik tok recomendando o app pra deixar o visual do pc mais bonito porem eu gostaria de saber se o app tem a funçao de fazer a troca automatica dos papeis de parede digo isso porque em meu pc eu tenho um app que eu acho que nao é muito conhecido porem ele é excelente na questao de trocar automaticamente imagens que eu nunca vi na vida mas sao boas e ainda da a opçao de voce escolher a categoria de imagens

**Comment by rocksdanister** [comment]
Currently no, its under development: https://github.com/rocksdanister/lively/issues/137

**Reply by tadkn** [reply]
alguma previsao de quando chega o recurso?

**Reply by rocksdanister** [reply]
Lively does not have a strict schedule, right now its planned for v2.1 release.

**Reply by tadkn** [reply]
> Lively does not have a strict schedule, right now its planned for v2.1 release.

obrigado pela informaçao mas voce sabe quando vai lançar essa versao?

**Comment by digaopk** [comment]
tambem gostaria saber quando vai fazer rodizio dos wallpapers , vai ficar muito legal
